paper/basic_training.tex

```diff
@@ -730,11 +731,170 @@ \subsubsection{Summary}
+the system will interact with the system enclosure. These impacts from
+the particles on the "enclosure" will impart a stress on the system box
+which can be related to the stress the surroundings are imparting on
+the system. With this relationship, we can use the virial theorem to
+calculate the pressure that the system is experiencing\cite{ShellNotes}.
+However, this is much more challenging when considering pairwise
+interactions and periodic boundary
+conditions\cite{allenTildesleyLiquids, tuckermanBook, ShellNotes}.
+A different approach to the virial theorem is necessary at that point.
+The math is a bit more complex, which is out of the scope of this
+article. We encourage the readers to read the articles referenced
+here for more information.
+
+Finally, this treatment described above only covers holding the pressure
+constant (the NP of NPT). When a barostat is applied without an
+additional thermostating algorithm, only the amount of particles (N),
+the pressure (P), and the enthalpy (H) of the system is held constant.
+This is known as the isoenthalpic-isobaric ensemble (NPH).
+To sample from the isothermal-isobaric ensemble (NPT), a
+thermostating algorithm like the ones dicussed earlier must also
+be applied.
+
+If the thermostat section was any indication, there have been a
+multitude of various barostat algorithms that have been developed
+during the lifetime of molecular dynamics. However, they usually can
+be classified into three main categories: volume rescaling, weakly
+coupled, and extended ensemble barostats \cite{ShellNotes,
+tuckermanBook}. The next section will describe the main differences
+between these barostats, and give some recommendations for proper
+use.
+
+
+\subsubsection{Popular and Notable Barostats}
+Within this section, a few notable barostats will be introduced to the
+reader, describing a high-level summary of each, with some of the issues
+associated with them as well. This is not an exhaustive list of
+barostats and barostat algorithms, just a sampling of popular and
+historic ones used in MD.
+
+\subparagraph{Volume Rescaling}
+\begin{enumerate}[listparindent=\parindent]
+    \item \textbf{Volume Rescaling}
+
+Volume rescaling barostats are the simplest example of pressure
+control in molecular simulations. Every time this barostat is
+executed, the volume of the system is modified to produce the exact
+pressure desired. This does \textbf{not} sample the proper
+ensemble, this cannot be used for production
+sampling \cite{ShellNotes}. This also does not smoothly approach the
+target pressure either, which might cause very unphysical issues
+with the system during time integration and force calculation.
+
+\end{enumerate}
+
+\subparagraph{Weakly Coupled Barostats}
+\begin{enumerate}[listparindent=\parindent]
+\item \textbf{Berendsen}
+
+The Berendesen \cite{berendsen1984molecular} barostat is very
+similar to the Berendsen thermostat discussed earlier. With the
```

Comment:
"With the..." is not a complete sentence. Perhaps "It seeks to improve upon the volume..."